### PR TITLE
fixed capitalization of Minitest to downcase minitest

### PR DESCRIPTION
**Pull request by marlomajor**
Hey I've noticed the capitalization throughout the first half of the docs of the word "minitest" isn't capitalize, but in the other half of the docs it is capitalize. I have removed the capitalize "M" from the cases in which the downcase version was used in the first half. Is this something you're looking for? Should I upcase all of the "minitest" words throughout instead? Let me know! Excited to contribute.

### CODE REVIEW

**Comment by bhenderson** [comment]
hey @marlomajor 

the convention in ruby is for Modules/Classes to be capitalized.

**Comment by marlomajor** [comment]
Oh no...I just realized that by using Replace All it corrected the capitalization of all of the minitest words. What I meant to do was to change it when used as a noun in sentences. Such as the first sentence of the docs:

"minitest provides a complete suite of testing facilities supporting TDD, BDD, mocking, and benchmarking."

Should minitest in the above example be capitalized? Its in reference to the class Minitest and it is at the beginning of the sentence.

**Comment by zamith** [comment]
I would keep it capitalised everywhere. For both the Minitest class and the Minitest library.

**Comment by marlomajor** [comment]
Okay I have made the changes that you have requested. If I can contribute anything else or you would like something changed, please let me know!

File: README.rdoc [hunk 1]
```diff
@@ -1,14 +1,14 @@
-= minitest/{unit,spec,mock,benchmark}
+= Minitest/{unit,spec,mock,benchmark}
 
-home :: https://github.com/seattlerb/minitest
-bugs :: https://github.com/seattlerb/minitest/issues
-rdoc :: http://docs.seattlerb.org/minitest
-vim  :: https://github.com/sunaku/vim-ruby-minitest
-emacs:: https://github.com/arthurnn/minitest-emacs
+home :: https://github.com/seattlerb/Minitest
```
Comment (zamith) [inline README.rdoc]: All the urls should be downcased.

File: README.rdoc [hunk 2]
```diff
@@ -1,14 +1,14 @@
-= minitest/{unit,spec,mock,benchmark}
+= Minitest/{unit,spec,mock,benchmark}
```
Comment (zamith) [inline README.rdoc]: This refers to a directory name, so it should probably be unchanged.

File: README.rdoc [hunk 3]
```diff
@@ -16,32 +16,32 @@ TDD, BDD, mocking, and benchmarking.
...
 
-     I MUST say that minitest is *very* readable / understandable
+     I MUST say that Minitest is *very* readable / understandable
      compared to the 'other two' options we looked at. Nicely done and
      thank you for helping us keep our mental sanity."
 
     -- Wayne E. Seguin
 
-minitest/unit is a small and incredibly fast unit testing framework.
+Minitest/unit is a small and incredibly fast unit testing framework.
```
Comment (zamith) [inline README.rdoc]: Again, this refers to a directory.

**Comment by zamith** [comment]
Ok, there are a few places where we can be more consistent with the casing, but it cannot be done with a find and replace, you need to evaluate every situation and see if it makes sense to downcase or capitalize.

I believe that if you do so, then the changes will be in few and easier to accept as a PR.

**Comment by marlomajor** [comment]
Okay I'm on it.

**Comment by zenspider** [comment]
Sorry. I'm shutting this down. Blind global text substitution is not the type of contribution we are looking for. If you find any ambiguities, inaccuracies, or errors in any files, please let us know in a new issue.

**Pull request closed without merging**